app/models/episode.rb

```diff
@@ -84,7 +84,7 @@ def self.story_uri(story)
   end
 
   def apple?
-    categories_include?([APPLE_ADFREE_TAG, APPLE_ONLY_TAG].freeze)
+    categories_include?([APPLE_FREEMIUM_TAG, APPLE_ONLY_TAG].freeze)
```

Comment:
Maybe I already asked this ... but I'm still not entirely clear why we need these hardcoded "apple" tags?

Doesn't the `apple_credential.public_feed.filtered_episodes` and `apple_credential.private_feed.filtered_episodes` already tell us what content to send to apple?